examples/nimble_gnrc/ble.c

```diff
@@ -0,0 +1,364 @@
+
```

Comment:
This file could also go to `shell_commands`, couldn't it? I believe then (and with the dependency fixes proposed in the Makefile) we could get away with not duplicating `gnrc_networking` ;-).

Reply:
not really. The thing is, that these shell commands are quite application specific. I have some other code almost ready to be PRed, that does some sort of automated connection management (`nimble_autoconn` (connect to anything that fits some filter criteria) and `nimble_rble` (RPL-over-BLE)). Both use their own specific event callback implementations and would clash badly with these shell command implemenation here. So the shell command is not as generic as it seems...

Reply:
maybe `sc_nimble_manual_scan` then? You say yourself, that these commands has nothing to do with `nimble` or `gnrc_netif` themselves, but rather are for managing connections. So a potential for duplication is there while they do not contributing to examplify GNRC+Nimble (i.e. have not much to do with this example).